pkgs/development/libraries/mesa/default.nix

```diff
@@ -99,7 +99,7 @@ let
   # FIXME: these should really go into some sort of versioned LLVM package set
   rust-bindgen' = buildPackages.rust-bindgen.override {
     rust-bindgen-unwrapped = buildPackages.rust-bindgen.unwrapped.override {
-      clang = buildPackages.llvmPackages_15.clang;
+      clang = buildPackages.llvmPackages_16.clang;
```

Comment:
Would it make sense to just use the defaults now, and assume we're going to keep LLVM at a reasonable version from now on?

Reply:
I don't trust us to be able to bump it quickly yet, but let's try.